src/main/scala/scala/tools/refactoring/analysis/CompilationUnitDependencies.scala

```diff
@@ -447,6 +457,23 @@ trait CompilationUnitDependencies extends CompilerApiExtensions with ScalaVersio
                   fakeSelectTreeFromType(tpe, sym, t.pos) match {
                     case s: Select if !isDefinedLocallyAndQualifiedWithEnclosingPackage(s) =>
                       addToResult(s)
+                    case i: Ident if newWay =>
+                      val realText = i.pos.source.content.slice(i.pos.start, i.pos.end).mkString
+                      val classOfRegex = """(?<=classOf\[)(.+)(?=\])""".r
```

Comment:
Isn't this the same as `str.startsWith("classOf[") && str.endsWith("]")`? I think that is easier to read than the regex.